src/pages/api/mock-backend/private/me/media.ts

```diff
@@ -0,0 +1,33 @@
+/*
+ * SPDX-FileCopyrightText: 2022 The HedgeDoc developers (see AUTHORS file)
+ *
+ * SPDX-License-Identifier: AGPL-3.0-only
+ */
+
+import type { NextApiRequest, NextApiResponse } from 'next'
+import { HttpMethod, respondToMatchingRequest } from '../../../../../handler-utils/respond-to-matching-request'
+import type { MediaUpload } from '../../../../../api/media/types'
+
+/*
+ * SPDX-FileCopyrightText: 2022 The HedgeDoc developers (see AUTHORS file)
+ *
+ * SPDX-License-Identifier: AGPL-3.0-only
+ */
+const handler = (req: NextApiRequest, res: NextApiResponse) => {
+  respondToMatchingRequest<MediaUpload[]>(HttpMethod.GET, req, res, [
```

Comment:
`|| respondMethodNotAllowed(res)` missing

Reply:
Hm. maybe we could merge the two methods? We only use them together.. :thinking:

Reply:
Changed.